ratelimit/auths/jwt.py

```diff
@@ -18,6 +18,9 @@
 def create_jwt_auth(
     key: Union[bytes, str, RSAPublicKey, RSAPrivateKey],
     algorithms: Union[List[str], str],
+    user_key: str = "user",
+    group_key: str = "group",
+    default_group: str = "default"
```

Comment:
Group default value always must be `default`. Please remove the parameter.

Reply:
OK